src/main/java/com/mongodb/hibernate/internal/MongoAssertions.java

```diff
@@ -115,4 +115,19 @@ public static boolean assertFalse(boolean value) throws AssertionError {
         }
         return false;
     }
+
+    /**
+     * Asserts that {@code value} is {@code false} with message.
+     *
+     * @param message the message to explain details.
+     * @param value A value to check.
+     * @return {@code false}.
+     * @throws AssertionError If {@code value} is {@code true}.
+     */
+    public static boolean assertFalse(String message, boolean value) throws AssertionError {
```

Comment:
Actually, now we don't need this method anymore, do we?

Reply:
we don't need it in this PR. Just wonder whether we can retain it for future PR. We had counterpart of `assertTrue` already, so I think it is fine to retain it? Let me know I could delete it without issue.

Reply:
We do have `MongoAssertions.assertTrue(String message, boolean value)`, but it's not used. And it is unlikely that that method or the new `assertFalse` method it to be used, as that requires us to need to accompany the assertion error with runtime data in its message.

I propose removing both methods, because having them without a legitimate usage only increases the chances that they will be used to supply a useless static assertion message.

Reply:
removed both